pkg/version/version.go

```diff
@@ -129,6 +133,20 @@ func ClusterUrlByVersion(version string) (string, error) {
 	return fmt.Sprintf(newClusterYamlUrl, Version), nil
 }
 
+func ResourceQuotUrlByVersion(version string) (string, error) {
+	lessThanOrEqual, err := VersionIsLessThanOrEqual(version, ClusterOperatorLastVersion())
+	if err != nil {
+		return "", err
+	}
+	if lessThanOrEqual {
+		return "", nil
+	}
+
+	// resource-quota.yaml is located on plugin repo,
+	// so we use 'Version' (plugin) instead of 'version' (operator).
+	return fmt.Sprintf(resourceQuotaYamlUrl, version), nil
```

Comment:
should be `Version` right? 
Version/version should probably have clearer names, but that's for a future PR I think.